portfolio/src/test/java/com/google/sps/BookParserTestFile.csv

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
You could use `.json` as file extension. `.csv` is a very specific file format "Comma Separated Values".